doc/iterations/2023.6/scopes/scope Migration flow.adoc

```diff
@@ -0,0 +1,17 @@
+= Scope
+
+== First task
+
+- [] Shape written, reviewed and merged
+
+== Second task
+
+- [] First prototype of implementation
+- [] Improvement of the first prototype after the discovery of performance issues
+- [] Second prototype of the backend merged
+
+
+== Third task
+
+- [ ] User guide updated with screenshots
+- [ ] Integrated tests updated
```

Comment:
As mentionned in your other shape pull request, do not contribute the scope now. It is to be maintained in your branch where you will work on the feature and contributed when it's over.
Same comment regarding the title of the commit / pull request